lib/src/components/NxTable/NxTableCell.tsx

```diff
@@ -96,10 +96,11 @@ const NxTableCell = function NxTableCell(props: NxTableCellProps) {
   return (
     <Tag className={classes} aria-sort={ariaSort} {...attrs}>
       {
-        isSortable ? cellSortingContents :
-        rowBtnIcon ? rowBtnCellContents :
-        children
-      }
+          isSortable ? cellSortingContents :
+          rowBtnIcon ? rowBtnCellContents :
+          children
+        }
+      {isHeader && rowBtnIcon && <span>Select Row</span>}
```

Comment:
Probably better to give this span a class and use that in the CSS

Reply:
> Probably better to give this span a class and use that in the CSS

Changed here: 8797e5690fea8222b4fc812549cc3e7ad319ced6

Reply:
> Changed here: https://github.com/sonatype/sonatype-react-shared-components/commit/8797e5690fea8222b4fc812549cc3e7ad319ced6

Again, classes should be about what the thing is not how it looks.  So I was expecting something like `nx-cell__row-btn-header-text` as the name here.